plugins/woocommerce-blocks/tests/e2e/tests/templates/constants.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * External dependencies
+ */
+import type { Page, Response } from '@playwright/test';
+import type { FrontendUtils } from '@woocommerce/e2e-utils';
+
+/**
+ * Internal dependencies
+ */
+import { SIMPLE_VIRTUAL_PRODUCT_NAME } from '../checkout/constants';
+import { CheckoutPage } from '../checkout/checkout.page';
+
+type TemplateCustomizationTest = {
+	visitPage: ( props: {
+		frontendUtils: FrontendUtils;
+		page: Page;
+	} ) => Promise< void | Response | null >;
+	templateName: string;
+	templatePath: string;
+	templateType: string;
+	defaultTemplate?: {
+		templateName: string;
+		templatePath: string;
+	};
+};
+
+export const CUSTOMIZABLE_WC_TEMPLATES: TemplateCustomizationTest[] = [
```

Comment:
While this constant is currently only used in one file (`template-customization.block_theme.side_effects.spec.ts`), my plan is to update `template-user-customization.block_theme_with_templates.spec.ts` and `template-user-customization.block_theme.spec.ts` to use it as well.